### ISSUE TITLE
Add a way to update records based on non-primary unique column

### ISSUE THREAD

**Issue by Destroy666x** [DBDiff/DBDiff#88]
As the title says. In some applications you don't mind primary keys changing and you have other fields that make the records unique.

**Comment by zenomordiante** [comment]
Yes, it's a good Idea. I have some problems with table with unique key.

**Comment by nkboscia** [comment]
Yes, many tables do not have a primary key, so we can't use this tool. :(  That should be stated in the prerequisites so people know before trying to use it.
